source/assoc.js

```diff
@@ -22,6 +23,12 @@ import _curry3 from './internal/_curry3';
  *      R.assoc('c', 3, {a: 1, b: 2}); //=> {a: 1, b: 2, c: 3}
  */
 var assoc = _curry3(function assoc(prop, val, obj) {
+  if (_isInteger(prop) && _isArray(obj)) {
+    var arr = [].concat(obj);
```

Comment:
I know that this was moved from elsewhere in the code, but still, I can see no reason for `[].concat(obj)` over `[obj]`.

Reply:
> I know that this was moved from elsewhere in the code, but still, I can see no reason for `[].concat(obj)` over `[obj]`.

I‘m not sure if I understand your meaning correctly. 

```js
var obj = [1, 2, 3]

// `[].concat(obj)` is cloning a array shallowly, 
;[].concat(obj) // => [1, 2, 3]

// [obj] can not
;[obj] // => [1, 2, 3]
```

Reply:
Never mind.  Brain fart.